portfolio/src/main/webapp/scripts/comments.js

```diff
@@ -12,11 +12,51 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/* Create a new HTML element */
+/* Function called on page load to show comments and check if the user can comment */
+async function init() {
+  setLoginLogoutURL();
+  getComments();
+
+  isLoggedIn().then(email => { // allow user to comment if logged in
+    console.log(email)
+    if (email) {
+      document.getElementById("comment-form").style.display = "flex";
+      document.getElementById("comment-blocker").style.display = "none";
+    }
+  })
+}
+
+/* Helper function to check if the user is logged in */
+async function isLoggedIn() {
```

Comment:
The current function name seems to imply that this function will return a boolean.

Reply:
Changed to getUserEmail()